app/templating/template_renderer.py

```diff
@@ -15,11 +15,13 @@ def __init__(self):
 
         self.environment.filters['format_date'] = format_date
```

Comment:
Following this up from my last comment; I see we already have decorators for these in the jinja_filters module. Do we know why we're effectively duplicating the assignment to the environment here?

Reply:
they are in the template renderer, so that we can use them in the schema for piped currencies

Reply:
But they are also decorated. See [Flasks docs](http://flask.pocoo.org/docs/0.12/templating/#registering-filters). My reading of that is that we shouldn't be doing both.

Reply:
Yes but they are also decorated in the jinja_filters module. See (Flasks docs)[http://flask.pocoo.org/docs/0.12/templating/#registering-filters]. My reading of that is they're equivalent and so we shouldn't be doing both?

Reply:
Direct assignment is used as TemplateRenderer has its own Jinja environment while the decorators are used for the rest of the application.

TemplateRenderer is a bit of a misnomer as it doesn't render templates, it does variable substitution on dicts (usually dicts populated by json.load) and strings, returning the resulting dict or string.

There is a card on the Tech board to re-evaluate this approach and tackling it now is beyond the scope of this PR.